docker/federation/docker-compose.yml

```diff
@@ -11,29 +11,29 @@ services:
       # Hack to make this work from both docker/federation/ and docker/federation-test/
       - ../federation/nginx.conf:/etc/nginx/nginx.conf
     depends_on:
-      - lemmy_alpha
-      - pictrs_alpha
-      - lemmy_beta
-      - pictrs_beta
-      - lemmy_gamma
-      - pictrs_gamma
+      - lemmy-alpha
+      - pictrs-alpha
+      - lemmy-beta
+      - pictrs-beta
+      - lemmy-gamma
+      - pictrs-gamma
```

Comment:
so technically underscores aren't valid and trust-dns-resolver (used by actix-web's client) actually cares about that